components/Flipper/FlipperFilter/FlipperFilter.tsx

```diff
@@ -179,170 +133,262 @@ function FlipperFilter(props: Props) {
+                            <Form.Label htmlFor="advancedCheckbox" className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>
+                                Advanced
+                            </Form.Label>
+                        }
+                        tooltipContent={<span>Get more advanced config options. (You shouldn't need them by default)</span>}
+                    />
+                    <Form.Check
+                        id="advancedCheckbox"
+                        onChange={e => {
+                            setIsAdvanced(e.target.checked)
+                        }}
+                        className={styles.flipperFilterFormfield}
+                        type="checkbox"
+                    />
+                </Form.Group>
+            </div>
+            {isAdvanced ? (
+                <>
+                    <hr />
+                    <div className={styles.flipperFilterGroupAdvanced}>
+                        <Form.Group className={styles.filterTextfield}>
+                            <Form.Label htmlFor="min-profit-percent" className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>
+                                Min. Profit (%):
+                            </Form.Label>
+                            <NumberFormat
+                                id="min-profit-percent"
+                                onValueChange={value => {
+                                    onSettingsChange('minProfitPercent', value.floatValue || 0)
+                                }}
+                                className={`${styles.flipperFilterFormfield} ${styles.flipperFilterFormfieldText}`}
+                                disabled={!props.isLoggedIn}
+                                isAllowed={value => {
+                                    return numberFieldMaxValue(value.floatValue, 2147483647)
+                                }}
+                                customInput={Form.Control}
+                                defaultValue={flipperFilter.minProfitPercent}
+                                thousandSeparator={getThousandSeperator()}
+                                decimalSeparator={getDecimalSeperator()}
+                                allowNegative={false}
+                                decimalScale={0}
+                            />
+                        </Form.Group>
+                        <Form.Group className={styles.filterTextfield}>
+                            <Tooltip
+                                type="hover"
+                                content={
+                                    <Form.Label htmlFor="min-volume" className={`${styles.flipperFilterFormfieldLabel} ${styles.checkboxLabel}`}>
+                                        Min. Volume:
+                                    </Form.Label>
+                                }
+                                tooltipContent={<span>Minimum amount of sold auctions in the last 24 hours</span>}
```

Comment:
```suggestion
                                tooltipContent={<span>Minimum average amount of sells in 24 hours</span>}
```
its mostly not the last 24 hours but longer/shorter timespans